<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <title>Welcome to Zhong Lab of Systems Biology</title>
	<!-- core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/font-awesome.min.css" rel="stylesheet">
    <link href="css/animate.min.css" rel="stylesheet">
    <link href="css/owl.carousel.css" rel="stylesheet">
    <link href="css/owl.transitions.css" rel="stylesheet">
    <link href="css/prettyPhoto.css" rel="stylesheet">
    <link href="css/main.css" rel="stylesheet">
    <link href="css/responsive.css" rel="stylesheet">
    <!--[if lt IE 9]>
    <script src="js/html5shiv.js"></script>
    <script src="js/respond.min.js"></script>
    <![endif]-->
    <link rel="shortcut icon" href="images/ico/z-icon-16px-16px.png">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="images/ico/apple-touch-icon-ipad-retina-152x152.png">
    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="images/ico/apple-touch-icon-iphone-retina-120x120.png">
    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="images/ico/apple-touch-icon-ipad-76x76.png">
    <link rel="apple-touch-icon-precomposed" href="images/ico/apple-touch-icon-ipad-76x76.png">
</head><!--/head-->

<body id="home" class="homepage">

    <header id="header">
        <nav id="main-menu" class="navbar navbar-default navbar-fixed-top" role="banner">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                    <a class="navbar-brand" href="index.html"><img src="images/logo3.png" alt="logo"></a>
                </div>

                <div class="collapse navbar-collapse navbar-right">
                    <ul class="nav navbar-nav">
                        <li class="scroll active"><a href="#home">Home</a></li>
                        <li class="scroll"><a href="#projectlist">Projects</a></li>
                        <li class="scroll"><a href="#portfolio">Publications</a></li>
                        <li class="scroll"><a href="#services">Software</a></li>
                        <!-- <li class="scroll"><a href="#about">About</a></li> -->
                        <li class="scroll"><a href="./team.html">Team</a></li>
                        <!-- <li class="scroll"><a href="#pricing">Pricing</a></li> -->
                        <!-- <li class="scroll"><a href="#blog">Blog</a></li> -->
                        <li class="scroll"><a href="#courselist">Courses</a></li>
                        <li class="scroll"><a href="#cta2">Join us</a></li>
                        <li class="scroll"><a href="#get-in-touch">Contact</a></li>
                    </ul>
                </div>
            </div><!--/.container-->
        </nav><!--/nav-->
    </header><!--/header-->

    <!--section id="main-slider">
        <div class="owl-carousel">
             <div class="item" style="background-image: url(images/slider/currbio-1600px-550px.png);">
                <div class="slider-inner">
                    <div class="container">
                        <div class="row">
                            <div class="col-sm-6">
                                <div class="carousel-content">
                                    <h2> Mapping <span> RNA-DNA </span> interactions</h2>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

            <div class="item" style="background-image: url(images/slider/genomeres-2014-1600px-550px-1.png);">
                <div class="slider-inner">
                    <div class="container">
                        <div class="row">
                            <div class="col-sm-6">
                                <div class="carousel-content">
                                    <a href="http://genome.cshlp.org/content/24/11/1787" target="_blank"><h2 style="color:#000"> Single cell <span> analysis of embryonic development </span> </h2></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

            <div class="item" style="background-image: url(images/slider/pnas-clustering-1600px-550px.png);">
                <div class="slider-inner">
                    <div class="container">
                        <div class="row">
                            <div class="col-sm-6">
                                <div class="carousel-content">
                                    <h2>Time-course clustering</h2>
                                    <p> </p>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

            <div class="item" style="background-image: url(images/slider/gr-epigenome-flower-1600px-550px.png);">
                <div class="slider-inner">
                    <div class="container">
                        <div class="row">
                            <div class="col-sm-6">
                                <div class="carousel-content">
                                    <a href="http://genome.cshlp.org/content/23/2/352.abstract" target="_blank"><h2 style="color:#000"> Spatio-temporal <span> clustering  of epigenome </span> </h2></a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

        </div>
    </section-->
    
    <section id="cta2">
        <div class="text-center">
                <h2 class="wow fadeInUp" data-wow-duration="300ms" data-wow-delay="0ms"><br><span>JOIN</span> US TO REVERSE AGING</h2>
        </div>
        <div class="container">
            <div class="text-center">
                <!--p class="wow fadeInUp" data-wow-duration="300ms" data-wow-delay="100ms">We welcome postdocs and graduate students who love science and dare to invent.</p-->
                <!-- p class="wow fadeInUp" data-wow-duration="300ms" data-wow-delay="200ms"><a class="btn btn-primary btn-lg" href="#">Free Download</a></p -->
                <img class="img-responsive wow fadeIn" src="images/cta2/cta2-img.png" alt="" data-wow-duration="300ms" data-wow-delay="300ms">
            </div>
        </div>
    </section>

    <section id="features">
        <div class="container">
            <div class="section-header">

            	<!--div class="section-header"-->
                <h2 class="section-title text-center wow fadeInDown">Postdocs TO REJUVENATE the AGING BRAIN</h2>

				<p class="text-left wow fadeInDown">We are recruiting audacious postdocs to rejuvenate the aging brain. We aim to transform the diagnoses and develop cures for Alzheimer’s disease. 
				Our approach is to iteratively improve interpretable artificial intelligence (AI), 
				technology, big data generation, and experimental and clinical validations. 
				We are recruiting bioinformatics postdocs to develop interpretable AI and neuroscience postdocs to evaluate AI-identified candidate routes. </p> <br>
				
				<h2 class="section-title text-center wow fadeInDown">
				Postdocs to create transformative technologies</h2>

				<p class="text-left wow fadeInDown">We welcome bold minds who will innovate in genomic technologies, including spatial, single-cell, and extracellular transcriptomics. 
				We are interested in revealing novel RNA types and functions. Our favorite application areas include blood vasculature in diverse tissues and diseases, liquid biopsy-based diagnoses, and mechanisms of 
				neurodegeneration and aging. 
				Ph.D.s in genomics, bioinformatics, molecular biology, and neuroscience are most welcome. </p> <br>  
 
				<!--/div-->

                <h2 class="section-title text-center wow fadeInDown">About the PI</h2>

				<p class="text-left wow fadeInDown">Sheng Zhong is a Professor of Bioengineering at UC San Diego. 
				He received an NIH Director's Pioneer Award, an NIH Catalyst Award in Diabetes, Endocrinology and Metabolic Diseases, an NIH Director's New Innovator Award, an NSF CAREER Award, and an Alfred Sloan Fellowship. 
				He serves as the director for <a href="https://biopsy.ucsd.edu/">UCSD Center for Liquid Biopsy Research</a> and the organizational hub of the NIH funded <a href="https://www.4dnucleome.org/">4D Nucleome (4DN)</a> Network. 
				He leads a Human Cell Atlas (HCA) seed network and a Transformative Technology Development team for the Human BioMolecular Atlas Program (HuBMAP). 
				Eight of his previous trainees are contributing to science on tenure-track faculty positions.</p><br>  

                <h2 class="section-title text-center wow fadeInDown">About the lab</h2>
				
				<p class="text-left wow fadeInDown">Our lab invented PROPER-seq (Protein-protein interaction by sequencing) to massively reveal protein-protein interactions (Molecular Cell, 2021). 
				We also invented the MARIO (Mapping RNA interactome in vivo) technology to massively reveal RNA-RNA interactions from human tissue (Nat Comm, 2016) and the MARGI (Mapping RNA-Genome Interactions) technology for revealing thousands of chromatin-associated RNAs (caRNA) and their respective genomic interaction sites (Current Biology, 2017; Nature Protocols, 2019). 
				Leveraging MARGI, we and our collaborators characterized caRNA’s roles in 3D organization of the nucleus (bioRxiv, 2021), modulation of gene expression during progression of diabetes mellitus in blood vessel endothelium (Nat Comm, 2020), and the biogenesis of fusion RNAs (PNAS 2019a). 
				These results inspired the idea that caRNA is a layer of the epigenome (Trends Genetics, 2018).</p> 

				<p class="text-left wow fadeInDown">We contributed to discovering the nuclear-encoded RNAs that are stably attached to the cell surface and exposed to the extracellular space, called membrane-associated extracellular RNAs (maxRNAs). 
				maxRNAs are functional components of the cell surface and mediate cell-cell interactions (Genome Biology, 2020).</p>

				<p class="text-left wow fadeInDown">We developed SILVER-seq for extracellular RNA (exRNA) sequencing from ultra-small volumes of liquid biopsy, solidifying a basis for future in vitro diagnostic trials using finger-prick blood for cancer screening (PNAS, 2019b) and diagnosis of Alzheimer's disease (Current Biology, 2020).</p>

				<p class="text-left wow fadeInDown">We contributed to discovering that the earliest cell fate decision in mouse is made sooner than the commonly thought 8-cell stage (Genome Res, 2014). 
				Our Rainbow-seq technology combined tracing of cell division history and single-cell RNA sequencing into one experiment (iScience 2018b). </p> 
				
				<p class="text-left wow fadeInDown">We contributed to revealing that transposons are indispensable regulatory sequences in the mammalian genomes. 
				Species-specific transposons are required for preimplantation embryonic development in humans and other mammals (Genome Res, 2010). 
				Nature highlighted this discovery as "Hidden Differences," reporting that "transposons or 'jumping genes' had hopped in front of the genes, changing their regulation" (Nature, 2010). 
				We contributed to establishing the proof-of-principle that cis-regulatory sequences can be annotated by cross-species epigenomic comparison (Cell, 2012). </p> 
            
            	<p>See more from our <a href="https://www.youtube.com/channel/UC60hDWkDav8RpPVeT2lbWYg">YouTube channel</a>.</p> 
            	<p>Read about <a href="https://doi.org/10.1242/bio.058656">Aitana Manuela Castro Colabianchi</a> who recently joined us as a postdoc scholar on Biology Open.</p>

            </div>
        </div>
    </section>
    
     <section id="projectlist">
        <div class="container">
            <div class="section-header">

            	<div class="section-header">
                <h2 class="section-title text-center wow fadeInDown">Projects for graduate rotations and extremely motivated undergraduates</h2>

				<p class="text-left wow fadeInDown"> Our projects contribute to the Human Cell Atlas (HCA), the Human BioMolecular Altas (HuBMAP), the 4D Nucleome (4DN) program, 
				diabetes research, and Alzheimer's research.</p>  

				</div>

                <h3 class="column-title text-left wow fadeInDown">Spatial transcriptomics with ultra-high spatial resolution </h3>

				<p class="text-left wow fadeInDown">Spatial transcriptomics methods allow for sequencing RNA from human tissue while registering the spatial locations of every sequenced RNA in the original tissue. 
				This project will develop next generation spatial transcriptomics tools that will significantly improve the spatial resolution and the size of the tissue that can be analyzed. 
				We will develop extremely high-resolution, high-density, high-fidelity and reproducible spatial mapping slides (HiFi slides) for spatial transcriptome analysis.</p> <br>  

                <!--h3 class="column-title text-left wow fadeInDown">Revealing protein-protein interactions at the genomic scale </h3>

				<p class="text-left wow fadeInDown">Genome-scale mapping of protein-protein interactions (PPI) remains laborious and resource-intensive. 
				This project will develop an extremely high-throughput genomic-based technology for mapping the human PPI network at the genomic scale. 
				This genomic technology and its coupled genomic informatics tools will generate a reference map of the human PPI network.</p> <br--> 

                <h3 class="column-title text-left wow fadeInDown">Revealing drug-protein interactions between a drug library and all human proteins by one experiment </h3>

				<p class="text-left wow fadeInDown">This project will develop a genomic technology that can reveal all drug-protein interactions between a small molecule library and a protein library. 
				This technology is expected to transform the current practices of identification of drug screening by 1000-fold increase of efficiency.</p> <br> 
				
				<h3 class="column-title text-left wow fadeInDown">Developing interpretable AI for diagnosis of Alzheimer's diease </h3>

				<p class="text-left wow fadeInDown">Develop interpretable AI for AD diagnosis based on big data of aging brain and blood plasma, including brain (single-nucleus) transcriptomes, epigenomics, imaging, blood extracellular RNA, cfDNA sequencing.</p>
				<br> 

                <!--h3 class="column-title text-left wow fadeInDown">Revealing RNA-chromatin interactions in single cells </h3>

				<p class="text-left wow fadeInDown">This project will develop technologies that can reveal RNA-chromatin interactions in single cells.</p> <br> 
				
				<h3 class="column-title text-left wow fadeInDown">The relationship between genome-wide RNA-chromatin interactions and 3D genome organization </h3>

				<p class="text-left wow fadeInDown">It remains unclear to what extent chromatin-associated RNAs can reflect the 3D organization of the genome. 
				To this end, we used iMARGI (a sequencing technology) to map genome-wide RNA-chromatin interactions in human embryonic stem cells, foreskin cells, and leukemia cells. 
				This project will compare these iMARGI data with genome interaction data including Hi-C and PLAC-seq on three different scales. 
				At the compartment scale, we will test whether the A compartment chromatin is associated with large amounts of RNAs, involving both intrachromosomal and interchromosomal RNA-chromatin interactions. 
				At the TAD scale, we will test whether the RNA ends of nearly all RNA-chromatin interactions are confined to within the boundaries of one or of a few consecutive TADs. 
				At the loop scale, we will test whether RNA-chromatin interactions are enriched with PLAC-seq derived enhancer-promoter interactions.</p--> 

                </div>
       </div>
    </section>


	<section id="portfolio">
        <div class="container">
            <div class="section-header">
                <h2 class="section-title text-center wow fadeInDown">PUBLICATIONS</h2>
	            <p class="text-center wow fadeInDown">Complete list of publications on <a href="https://scholar.google.com/citations?hl=en&amp;user=EztoKEIAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" target="_blank">Google Scholar</a>, <a href="https://www.ncbi.nlm.nih.gov/myncbi/sheng.zhong.1/bibliography/public/" target="_blank">NCBI</a></p>
            </div>
            <!--div class="text-center" -->
                    <h3 class="column-title wow fadeInDown">Selected papers (please choose a category)</h3>

                    <div role="tabpanel">
                        <ul class="nav main-tab nav-justified" role="tablist">
                            <li role="presentation" class="active">
                                <a href="#RNA-tab" role="tab" data-toggle="tab" aria-controls="RNA-tab" aria-expanded="true">Interactome</a>
                            </li>
                            <li role="presentation">
                                <a href="#Liquidbiopsy-tab" role="tab" data-toggle="tab" aria-controls="Liquidbiopsy-tab" aria-expanded="false">Liquid biopsy, AD</a>
                            </li>
                            <li role="presentation">
                                <a href="#Epigenome-tab" role="tab" data-toggle="tab" aria-controls="Epigenome-tab" aria-expanded="false">Embryo, Chromatin</a>
                            </li>
                            <!--li role="presentation">
                                <a href="#Obesity-tab" role="tab" data-toggle="tab" aria-controls="Obesity-tab" aria-expanded="true">Metabolism</a>
                            </li-->

                            <li role="presentation">
                                <a href="#Evol-tab" role="tab" data-toggle="tab" aria-controls="Evol-tab" aria-expanded="false">AI, Aging</a>
                            </li>
                            <!--li role="presentation">
                                <a href="#Statistics-tab" role="tab" data-toggle="tab" aria-controls="Statistics-tab" aria-expanded="false">Statistics</a>
                            </li-->
                            <!--li role="presentation">
                                <a href="#Software-tab" role="tab" data-toggle="tab" aria-controls="Software-tab" aria-expanded="false">Software</a>
                            </li-->
                            <!--li role="presentation">
                                <a href="#Health-tab" role="tab" data-toggle="tab" aria-controls="Health-tab" aria-expanded="false">Public Health</a>
                            </li -->
                        </ul>

                        <div id="tab-content" class="tab-content">
                            <div role="tabpanel" class="tab-pane fade active in" id="RNA-tab" aria-labelledby="RNA-tab">
                                <ul class="text-left">
                                	<H4>Protein-protein interaction</H4>
                                	<li><b>Revealing protein-protein interactions at the transcriptome scale by sequencing.</b>
                            			Kara L. Johnson, Zhijie Qi, Zhangming Yan, Xingzhao Wen, Tri C.Nguyen, Kathia Zaleta-Rivera, Chien-JuChen, Xiaochen Fan, Kiran Sriram, Xueyi Wan, Zhen Bouman Chen, Sheng Zhong.<br>
                                        <b><u>Molecular Cell</u></b>, 2021, online preprint, <a href="https://doi.org/10.1016/j.molcel.2021.07.006" target="_blank">Text</a>, <a href="https://genemo.ucsd.edu/proper">PROPER database</a>, <a href="https://doi.org/10.5281/zenodo.5009171">PROPERTools software</a>,
                                        <a href="https://docs.google.com/document/d/1RnmKo-JFS-rH_LndAY_Ksm3krusjlgQOQVr81okehew/edit?usp=sharing" target="_blank">Interaction Ligation Linker</a>, 
									    <a href="https://youtu.be/OdfyycaxRS4" target="_blank">YouTube</a>.
									                                    
                                	<H4>RNA-chromatin interaction</H4>
                                    <li><b>Three-dimensional organization of chromatin associated RNAs and their role in chromatin architecture in human cells.</b>
                            			Riccardo Calandrelli, Xingzhao Wen, Tri C. Nguyen, Chien-Ju Chen, Zhijie Qi, Weizhong Chen, Zhangming Yan, Weixin Wu, Kathia Zaleta-Rivera, Rong Hu, Miao Yu, Yuchuan Wang, Jian Ma, Bing Ren, Sheng Zhong.<br>
                                        <b><u>bioRxiv</u></b>, 2021, 81(19):4091-4103. <a href="https://doi.org/10.1101/2021.06.10.447969" target="_blank">Text</a>, <a href="https://www.4dnucleome.org/" target="_blank"><img src="https://www.4dnucleome.org/Assets/images/4dn-logo_1.png" alt="4DN web portal" height="35"/></a>
                                    </li>

                                    <li><b>Stress-induced RNA-chromatin interactions promote endothelial dysfunction.</b>
                            			Riccardo Calandrelli, Lixia Xu, Yingjun Luo, Weixin Wu, Xiaochen Fan, Tri Nguyen, Chien-Ju Chen, Kiran Sriram, Xiaofang Tang, Andrew Burns, Rama Natarajan, Zhen Chen, Sheng Zhong.<br>
                                        <b><u>Nature Communications</u></b>, 2020, 11:5211. <a href="https://doi.org/10.1038/s41467-020-18957-w" target="_blank">Text</a>,
                                        <a href="https://youtu.be/w52ocO-IFjE" target="_blank">YouTube</a>.  
                                    </li>

                                    <li><b>Mapping RNA-chromatin interactions by sequencing with iMARGI.</b>
                            			Weixin Wu, Zhangming Yan, Tri C. Nguyen, Zhen Chen, Shu Chien, Sheng Zhong.<br>
                                        <b><u>Nature Protocols</u></b>, 2019, 14:3243–3272. <a href="https://doi.org/10.1038/s41596-019-0229-4" target="_blank">Text</a>, 
                                        <a href="https://www.protocols.io/view/mapping-rna-chromatin-interactions-by-sequencing-w-2cxgaxn" target="_blank">Protocol</a>, 
                                        <a href="https://sysbio.ucsd.edu/imargi_pipeline" target="_blank">Software</a>, <a href="https://www.4dnucleome.org/" target="_blank"><img src="https://www.4dnucleome.org/Assets/images/4dn-logo_1.png" alt="4DN web portal" height="35"/></a>  
                                    </li>

                                    <li><b>Genome-wide co-localization of RNA-DNA interactions and fusion RNA pairs.</b>
                            			Zhangming Yan, Norman Huang, Weixin Wu, Weizhogn Chen, Yiqun Jiang, Jingyao Chen, Xuerui Huang, Xingzhao Wen, Jie Xu, Qiushi Jin, Kang Zhang, Zhen Chen, Shu Chien, Sheng Zhong.<br>
                                        <b><u>PNAS</u></b>, 2019, 116 (8) 3328-3337. <a href="https://doi.org/10.1073/pnas.1819788116" target="_blank">Text</a>, <a href="https://youtu.be/KJG6BZL2tiI?t=1657" target="_blank">YouTube</a>, <a href="https://www.4dnucleome.org/" target="_blank"><img src="https://www.4dnucleome.org/Assets/images/4dn-logo_1.png" alt="4DN web portal" height="35"/></a> 
                                    </li>

                                    <li><b>RNA, action through interactions.</b>
                            			Tri C. Nguyen, Kathia Zaleta-Rivera, Xuerui Huang, Xiaofeng Dai, Sheng Zhong.<br>
                                        <b><u>Trends in Genetics</u></b>, 2018, 34:867-882. 
                                          <a href="https://doi.org/10.1016/j.tig.2018.08.001" target="_blank">Text</a>
                                    </li>
                                    
                                    <li><b>RNAs as proximity labeling media for identifying nuclear speckle positions relative to the genome.</b>
                            			Weizhong Chen, Zhangming Yan, Simin Li, Norman Huang, Xuerui Huang, Jin Zhang, Sheng Zhong.<br>
                                        <b><u>iScience</u></b>, 2018, 4:204-215.
                                          <a href="https://www.cell.com/iscience/fulltext/S2589-0042(18)30081-6" target="_blank">Text</a>, <a href="./images/iScience2018embryo.png">Cover proposal</a>
                                    </li>

                                    <li><b>Systematic mapping of RNA-chromatin interactions in vivo.</b>
                            			Bharat Sridhar, Marcelo Rivas-Astroza, Tri C. Nguyen, Weizhong Chen, Zhangming Yan, Xiaoyi Cao, Lucie Hebert, Sheng Zhong.<br>
                                        <b><u>Current Biology</u></b>, 2017, 27(4): 602–609.
                                          <a href="http://www.cell.com/current-biology/fulltext/S0960-9822(17)30011-8" target="_blank">Text</a>, 
                                          <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE92345" target="_blank">Data</a>, <a href="./publication/margi">Protocols</a>,  <a href="http://systemsbio.ucsd.edu/margi/" target="_blank">Bioinformatic pipeline</a>, <a href="http://f1000.com/prime/727251916?bd=1" target="_blank">F1000Prime</a>
                                    </li>
                                    
                                    <H4>RNA-RNA Interaction</H4>
                                    <li> <b>Mapping RNA-RNA interactome and RNA structure in vivo by MARIO.</b>
                            			Tri C. Nguyen, Xiaoyi Cao, Pengfei Yu, Shu Xiao, Jia Lu, Fernando H. Biase, Bharat Sridhar, Norman Huang, Kang Zhang, Sheng Zhong.<br> 
                            				<b><u>Nature Communications</u></b>, 2016, 7:12023.
                                          <a href="http://www.nature.com/ncomms/2016/160624/ncomms12023/full/ncomms12023.html" target="_blank">Text</a>, <a href="http://mariotools.ucsd.edu/" target="_blank">Software</a>,
                                          <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61489" target="_blank">Data</a>
                                    </li>

                                </ul>
                            </div>
                            <div role="tabpanel" class="tab-pane fade" id="Liquidbiopsy-tab" aria-labelledby="Liquidbiopsy-tab">
                                <ul class="text-left">
                                
                                	<H4>Cell Surface RNA</H4>
                            		<li><b>Natural display of nuclear-encoded RNA on the cell surface and its impact on cell interaction.</b>
                            			Norman Huang, Xiaochen Fan, Kathia Zaleta-Rivera, Tri C. Nguyen, Jiarong Zhou, Yingjun Luo, Jie Gao, Ronnie H. Fang, Zhangming Yan, Zhen Bouman Chen, Liangfang Zhang, Sheng Zhong.<br>
                                        <b><u>Genome Biology</u></b>, 2020, 21:225. <a href="https://doi.org/10.1186/s13059-020-02145-6" target="_blank">Text</a>, 
                                        <a href="https://youtu.be/rDnh91mSjuQ" target="_blank">YouTube</a>, 
                                        <a href="https://www.youtube.com/watch?v=Uw3_G69hgok" target="_blank">A 2-minute introduction</a>.   
                                    </li>
                                    
                                    <H4>Extracellular RNA</H4>
                                    <li><b>Extracellular RNA in a single droplet of human serum reflects physiologic and disease states.</b>
                            			Zixu Zhou, Qiuyang Wu, Zhangming Yan, Haizi Zheng, Chien-Ju Chen, Yuan Liu, Zhijie Qi, Riccardo Calandrelli, Zhen Chen, Shu Chien, H. Irene Su, Sheng Zhong.<br>
                                        <b><u>PNAS</u></b>, 2019, 116:19200–19208. <a href="https://www.pnas.org/content/116/38/19200" target="_blank">Text</a>, 
                                        <a href="https://docs.google.com/document/d/1Kdv7SFKEvFBaE4yDjERNhFL5G6Kvv7tfNr1wfTYLlak/edit?usp=sharing" target="_blank">SILVER-seq flowchart</a>, 
                                        <a href="https://github.com/Zhong-Lab-UCSD/MapSilverSeqDataWithExceRpt/blob/master/README.md" target="_blank">Mapping by exceRpt software</a>, 
                                        <a href="https://www.rnadiagnosis.com/" target="_blank">SILVER-seq service at Genemo</a>, <a href="https://clinicaltrials.gov/ct2/show/NCT01197456" target="_blank">The POISE trial @ ClinicalTrials.gov</a>.    
                                    </li>
                                    
                                    <li><b>Presymptomatic Increase of an Extracellular RNA in Blood Plasma Associates with the Development of Alzheimer’s Disease.</b>
                            			Zhangming Yan, Zixu Zhou, Qiuyang Wu, Zhen Bouman Chen, Edward H. Koo, Sheng Zhong.<br>
                                        <b><u>Current Biology</u></b>, 2020, 30:1771–1782. <a href="https://doi.org/10.1016/j.cub.2020.02.084" target="_blank">Text</a>, <a href="https://youtu.be/VuzAnWuhEto" target="_blank">Seminar</a> for the Extracellular RNA Communication (ERCC) consortium, 
                                        <a href="https://docs.google.com/document/d/1Kdv7SFKEvFBaE4yDjERNhFL5G6Kvv7tfNr1wfTYLlak/edit?usp=sharing" target="_blank">SILVER-seq flowchart</a>, 
                                        <a href="https://www.rnadiagnosis.com/" target="_blank">exRNA sequencing service at Genemo</a>, <a href="https://youtu.be/y99TuXCc59E?t=2388" target="_blank">Presentation in Chinese</a>.   
                                    </li>
                                    
                                    <H4>Alzheimer's disease</H4>
                                    <li><b>PHGDH expression increases with progression of Alzheimer’s disease pathology and symptoms.</b>
                            			Xu Chen, Riccardo Calandrelli, John Girardini, Zhangming Yan, Zhiqun Tan, Xiangmin Xu, Annie Hiniker, Sheng Zhong.<br>
                                        <b><u>Cell Metabolism</u></b>, 2022, In press. <a href="https://doi.org/10.1101/2022.02.15.480384">bioRxiv</a>. 
                                    </li>
                      
                                    <H4>Metabolism</H4>
                                    <li><b>Suppression of endothelial AGO1 promotes adipose tissue browning and improves metabolic dysfunction.</b>
                            			Xiaofang Tang, Yifei Miao, Yingjun Luo, Kiran Sriram, Zhijie Qi, Feng-Mao Lin, Kendall Van Keuren-Jensen, Patrick Fueger, Gene W. Yeo, Rama Natarajan, Sheng Zhong, Zhen Bouman Chen<br>
                                        <b><u>Circulation</u></b>, 2020, 142:365–379.<a href="https://doi.org/10.1161/CIRCULATIONAHA.119.041231">Article</a>, Editorial: <a href="https://doi.org/10.1161/CIRCULATIONAHA.120.047574">Endotheliopathy of Obesity</a>.    
                                    </li>
                                    <li><b>Genomic analysis of hepatic farnesoid X receptor binding sites reveals altered binding in obesity and direct gene repression by farnesoid X receptor in mice.</b>
                            			Jiyoung Lee, Sunmi Seok, Pengfei Yu, Kyungsu Kim, Zachary Smith, Marcelo Rivas‐Astroza, Sheng Zhong, Jongsook Kim Kemper<br>
                                        <b><u>Hepatology</u></b>, 2012, 56(1):108-117. <a href="https://doi.org/10.1002/hep.25609">Text</a>.    
                                    </li>

                                </ul>
                            </div>
                            <!--div role="tabpanel" class="tab-pane fade" id="Obesity-tab" aria-labelledby="Obesity-tab">
                                <ul class="text-left">

                                </ul>
                            </div-->

                            <div role="tabpanel" class="tab-pane fade" id="Epigenome-tab" aria-labelledby="Epigenome-tab">
                                <ul class="text-left">
                                
                            		<H4>Embryogenesis</H4>
                            		
                            		<li> <b>Rainbow-seq: combining cell lineage tracing with single-cell RNA sequencing in preimplantation embryos. </b>
                            			Fernando H. Biase, Qiuyang Wu, Riccardo Calandrelli, Marcelo Rivas-Astroza, Shuigeng Zhou, Zhen Chen, Sheng Zhong.<br>
                                    	<b><u>iScience</u></b>, 2018, 7:16-29. <a href="https://doi.org/10.1016/j.isci.2018.08.009" target="_blank">Text</a>
                                    	</li>

                            			<li> <b>Cell fate inclination within 2-cell and 4-cell mouse embryos revealed by single-cell RNA sequencing. </b>
                            			Fernando H. Biase, Xiaoyi Cao, Sheng Zhong. <br>
                                    	<b><u>Genome Research</u></b>, 2014, 24:1787-1796.
                                    	<a href="http://genome.cshlp.org/content/24/11.cover-expansion" target="_blank"><strong>Cover Article</strong></a>,
                                     	<a href="http://genome.cshlp.org/content/24/11/1787" target="_blank">Abstract</a>,
				  						<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57249" target="_blank">single-cell RNA-seq data</a>,
                                      	<a href="http://singlecell.cepbrowser.org/singlecell.php" target="_blank">single-cell genome browser</a>,
                                      	<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59892" target="_blank">single-cell Fluidigm qPCR data</a>,
                                      	<a href="http://systemsbio.ucsd.edu/singlecellped/" target="_blank">in situ images</a></li>

                            			<li> <b>Rewirable gene regulatory networks in the preimplantation embryonic development of three mammalian species. </b>
                            			Dan Xie, Chieh-Chun Chen, Leon M Ptaszek, Shu Xiao, Xiaoyi Cao, Fang Fang, Huck H Ng, Harris A Lewin, Chad Cowan, Sheng Zhong. <br>
                                    	<b><u>Genome Research</u></b>, 2010 20:804-815. <a href="http://genome.cshlp.org/content/20/6.cover-expansion" target="_blank"><strong>Cover Article</strong></a>,
                                    	<a href="http://genome.cshlp.org/content/20/6/804.abstract" target="_blank">Abstract</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18290" target="_blank"> Embryo data</a>,
                                    	<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18319" target="_blank"> Mtf2 knockdown data</a>. <br>
                                    	Research Highlight: Hidden differences. <a href="http://www.nature.com/nature/journal/v464/n7293/full/4641248b.html" target="_blank">Nature 464: 1248</a>.
                                    	</li>
                                    	
                                    	<li> <b>A core Klf circuitry regulates self-renewal of embryonic stem cells. </b>
                            			DJianming Jiang, Yun-Shen Chan, Yuin-Han Loh, Jun Cai, Guo-Qing Tong, Ching-Aeng Lim, Paul Robson, Sheng Zhong, Huck-Hui Ng. <br>
                                    	<b><u>Nature cell biology</u></b>, 2008, 10:353-360. <a href="https://doi.org/10.1038/ncb1698" target="_blank">Abstract</a>.
                                    	</li>
                                    	
                                    <H4>Chromatin</H4>
                            		
                            		<li><b>The 4D nucleome project.</b>
                            			 Job Dekker, Andrew S. Belmont,	Mitchell Guttman, Victor O. Leshyk,	John T. Lis, Stavros Lomvardas, Leonid A. Mirny, Clodagh C. O’Shea, Peter J. Park, Bing Ren, Joan C. Ritland Politz, Jay Shendure, Sheng Zhong &amp; the 4D Nucleome Network.<br>
                                    	<b><u>Nature</u></b>, 2017, 549:219–226. <a href="http://www.nature.com/nature/journal/v549/n7671/full/nature23884.html?WT.feed_name=subjects_genetics" target="_blank">Text</a>, <a href="./publication/4dn/4DNucleome2-layers-CMYK-small.png" target="_blank">Artwork</a>,
                                    	<a href="https://www.4dnucleome.org/" target="_blank"><img src="https://www.4dnucleome.org/Assets/images/4dn-logo_1.png" alt="4DN web portal" height="35"/></a>  
                                    </li>
									<li><b>SMARCAD1 contributes to regulation of naïve pluripotency by interacting with histone citrullination.</b>
                            			Shu Xiao, Jia Lu, Bharat Sridhar, Xiaoyi Cao, Pengfei Yu, Chieh-Chun Chen, Darina McDee, Laura Sloofman, Yang Wang, Marcelo Rivas-Astroza, Bhanu Prakash V.L. Telugu, Dana Levasseur, Kang Zhang, Han Liang, Jing Crystal Zhao, Tetsuya S. Tanaka, Gary Stormo, Sheng Zhong.<br>
                                    	<b><u>Cell Reports</u></b>, 2017, 18:3117-3128. <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(17)30288-7" target="_blank">Text</a>, <a href="http://sysbio.ucsd.edu/public/bharat/Smarcad1_raw_data/raw_data.html", target="_blank">Raw images</a>, 
                                    	<a href="http://jacobsschool.ucsd.edu/uploads/news_release/2017/SMARCAD1%20pinball%20art.jpg" target="_blank">Cover proposal</a>
                                    </li>

                            		<li><b>Spatiotemporal clustering of epigenome reveals rules of dynamic gene regulation.</b>
                            			Pengfei Yu, Shu Xiao, Xiaoyun Xin, Chun-Xiao Song, Wei Huang, Darina McDee, Tetsuya Tanaka, Ting Wang, Chuan He, Sheng Zhong. <br>
                                    	<b><u>Genome Research</u></b>, 2013, 23:352-384. <a href="http://genome.cshlp.org/content/23/2.cover-expansion" target="_blank"> <strong>Cover article</strong></a>,
                                    	<a href="http://genome.cshlp.org/content/23/2/352.abstract" target="_blank"> Abstract</a>,
                                    	<a href="GATE/" target="_blank">Software</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?targ=self&amp;form=html&amp;view=quick&amp;acc=GSE38596" target="_blank">Data</a>;
				  						<a href="http://bib.oxfordjournals.org/content/early/2013/11/06/bib.bbt078.abstract" target="_blank">Review</a>
                                    </li>

                            		<li><b>Understanding variation in transcription factor binding by modeling transcription factor genome-epigenome interactions.</b>
                            			 Chieh-Chun Chen, Shu Xiao, Dan Xie, Xiaoyi Cao, Chun-Xiao Song, Ting Wang, Chuan He, Sheng Zhong.<br>
                                    	<b><u>PLoS Computational Biology</u></b>, 2013, 9(12): e1003367.
                                          <a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1003367" target="_blank">
                                          Text</a>, <a href="apeg/" target="_blank">Software</a>, <a href="apeg/supp.html" target="_blank">Supplementary Figures</a>
                                    </li>
                                    
                                    <li><b>Comparative epigenomic annotation of regulatory DNA.</b>
                            			Shu Xiao, Dan Xie, Xiaoyi Cao, Pengfei Yu, Xiaoyun Xing, Chieh-Chun Chen, Meagan Musselman, Mingchao Xie, Franklin D. West, Harris A. Lewin, Ting Wang, Sheng Zhong.<br>
                                    	<b><u>Cell</u></b>, 2012, 49: 1381-1391.
                                          <a href="http://www.cell.com/cell/abstract/S0092-8674(12)00574-0" target="_blank">Abstract</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36114" target="_blank"> Data</a>, <a href="http://www.cepbrowser.org/" target="_blank">Comparative
                                            Epigenome Browser</a>. <br> Reviewed by: <a href="http://www.omicsonline.org/embryonic-stem-cells-a-perfect-tool-for-studying-mammalian-transcriptional-enhancers-2157-7633.S10-007.php?aid=7417" target="_blank">
                                        <em>J Stem Cell Res Ther</em></a>, 2012, S10:007.
                                        <a href="http://life.scichina.com:8082/sciCe/EN/abstract/abstract510383.shtml" target="_blank">
                                        <em>SCIENCE CHINA Life Sciences</em></a>, 2013, 56(3): 213-219.
                                        <a href="http://onlinelibrary.wiley.com/doi/10.1002/wsbm.1189/abstract" target="_blank"><em>WIREs Systems Biol Med</em></a>, 2012, 4(6): 525-545.
                                    </li>

                                    <li><b>The p23 molecular chaperone and GCN5 acetylase jointly modulate protein-DNA dynamics and open chromatin status.</b>
                            			 Elena Zelin, Yang Zhang, Oyetunji A Toogun, Sheng Zhong, Brian C Freeman.<br>
                                    	<b><u>Molecular Cell</u></b>, 2012, 48(3):459-470. <a href="https://doi.org/10.1016/j.molcel.2012.08.026" target="_blank">Text</a>.
                                    </li>
                                    
                                    <li><b>3D Genome: from technologies to visualization</b>. [<a href="https://zhonglab.gitbook.io/3dgenome/" target="_blank">ISBN: 987-1-17325643-0-5</a>]. eBook published by eScholarship. 
                                    Textbook for <a href="http://systemsbio.ucsd.edu/beng183fa2020/", target="_blank">BENG183</a>.
                                    </li>

                                </ul>
                            </div>

                            <div role="tabpanel" class="tab-pane fade" id="Evol-tab" aria-labelledby="Evol-tab">
                                <ul class="text-left">
                                    
                                    <H4>Aging</H4>
                                    <li><b>Cross-species microarray analysis with the OSCAR system suggests an INSR-Pax6-NQO1 neuro-protective pathway in ageing and Alzheimer's disease.</b> Yue Lu, Xin He, Sheng Zhong.<br>
                                          <b><u>Nucleic Acids Research</u></b>, 2007, 35: W105-W114. <a href="https://academic.oup.com/nar/article/35/suppl_2/W105/2924049/Cross-species-microarray-analysis-with-the-OSCAR" target="_blank">TEXT</a>.
                                    </li>

                                    
                                    <H4>Machine Learning</H4>
                                    <li><b>Time-variant clustering model for understanding cell fate decisions.</b>
                            			Wei Huang, Xiaoyi Cao, Fernando H. Biase, Pengfei Yu, Sheng Zhong.<br>
                                    	<b><u>PNAS</u></b>, 2014, 111(44):E4797-E4806.
                                          <a href="http://www.pnas.org/content/111/44/E4797.abstract" target="_blank">
                                          Abstract</a>
                                    </li>

                                    <li><b>Network based comparison of temporal gene expression patterns.</b>
                            			Wei Huang, Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>Bioinformatics</u></b>, 2010, 26(23): 2944-2951. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20889495" target="_blank">Abstract</a>, <a href="NACEP/" target="_blank">Software</a>
                                    </li>

                                    <li><b>Dissecting early differentially expressed genes in a mixture of differentiating embryonic stem cells.</b>
                                    Feng Hong, Fang Fang, Xuming He, Xiaoyi Cao, Hiram Chipperfield, Dan Xie, Wing H. Wong, Huck H. Ng, Sheng Zhong. <br>
                  					<b><u>PLoS Computational Biology</u></b>, 2009, 5(12): e1000607. <a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000607" target="_blank">Text</a>,
                  					<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE12550" target="_blank">Data</a>
                  					</li>

                  					<!--li><b>Gene Ontology analysis in multiple gene clusters under multiple hypothesis testing framework.</b>
	                                    Sheng Zhong and Dan Xie. <br>
	                  					<b><u>Artificial Intelligence in Medicine</u></b>, 2007, 41:105-115. <a href="http://www.aiimjournal.com/article/PIIS0933365707001005/abstract" target="_blank">Abstract</a>,
	                  					<a href="http://bioinformatics.bioen.illinois.edu/papers/GoSurfer-AIIM.pdf" target="_blank">PDF</a>.
	                  					This paper is reviewed by an <a href="http://www.aiimjournal.com/article/S0933-3657(07)00099-1/fulltext" target="_blank">Editorial</a> on <u>Artificial Intelligence in Medicine</u> 41:83-86.
                  					</li-->

                       				<li><b>Reproducibility Probability Score - incorporating measurement variability across laboratories for gene selection.</b>
	                                    Guixian Lin, Xuming He, Hanlee Ji, Leming Shi, Ronald Davis, Sheng Zhong.<br>
	                  					<b><u>Nature Biotechnology</u></b>, 2007, 41:105-115. 24(12): 6-7.
	                  					<a href="http://www.nature.com/nbt/journal/v24/n12/full/nbt1206-1476.html" target="_blank">Text</a>, <a href="rps/index.htm" target="_blank">Software</a>,
	                  					<a href="http://www.nature.com/nbt/journal/v24/n12/suppinfo/nbt1206-1476_S1.html" target="_blank">Supplementary Material</a>. The article has been reviewed by:
	                  					<a href="http://www.futuremedicine.com/doi/abs/10.2217/14622416.8.8.1037" target="_blank">Pharmacogenomics</a>, 2007, 8(8): 1037-1049.
										<a href="http://www.sciencedirect.com/science/article/pii/S1359634907700290" target="_blank"> European Journal of Cancer</a>, 2007, 5(5): 97-104.
										<a href="http://www.sciencedirect.com/science/article/pii/S0958166907001450" target="_blank">Current Opinion in Biotechnology</a>
										<a href="http://www.elsevier.com/books/systems-biomedicine/liu/978-0-12-372550-9" target="_blank">Systems Biomedicine: Concepts and Perspectives</a>, Edison Liu, Douglas Lauffenburger (editors), Elsevier, 2009, p.172.
										<a href="http://onlinelibrary.wiley.com/doi/10.1002/wsbm.151/abstract" target="_blank">WIREs Systems Biol Med</a>, 2012, 4(1): 39-49.
										<a href="http://onlinelibrary.wiley.com/doi/10.1002/wsbm.1189/abstract" target="_blank">WIREs Systems Biol Med</a>, 2012, 4(6): 525-545.
									</li>
									
									<H4>Evolution of Gene Regulatory Networks</H4>
									
									<li><b>EpiAlignment: alignment with both DNA sequence and epigenomic data.</b>
                            			Jia Lu, Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>Nucleic Acids Research</u></b>, 2019, 47(W1):W11-W19.
                                          <a href="https://doi.org/10.1093/nar/gkz426" target="_blank">Text</a>,
                                          <a href="https://epialign.ucsd.edu/" target="_blank">Software</a>. 
                                    </li>


                                     <li><b>A likelihood approach to testing hypotheses on the co-evolution of epigenome and genome.</b>
                            			Jia Lu, Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>PLoS Computational Biology</u></b>, 2018, 14(12):e1006673. 
                                    	<a href="https://doi.org/10.1371/journal.pcbi.1006673" target="_blank">Text</a>
                                    </li>
                                    
                                    <li><b>Towards an evolutionary model of transcription networks.</b>
                            			Dan Xie, Chieh-Chun Chen, Xin He, Xiaoyi Cao, Sheng Zhong.<br>
                                          <b><u>PLoS Computational Biology</u></b>, 2011, 7(6): e1002064. <a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1002064" target="_blank">Text</a>. <a href="EvoRgNet/" target="_blank">Website</a>.
                                    </li>

									<li><b>Modeling co-expression across species for complex traits: insights to the difference of human and mouse embryonic stem cells.</b>
                            			Jun Cai, Dan Xie, Zhewen Fan, John Marden, Wing H. Wong, Sheng Zhong.<br>
                                    	<b><u>PLoS Computational Biology</u></b>, 2010, 6(3): e1000707. <a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000707" target="_blank">Text</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE12550" target="_blank"> Data</a>, <a href="SCSC/tools/SCSC.html">S<span class="auto-style1">oftware</span></a>
                                    </li>

                            		<li><b>Cross-species de novo identification of cis-regulatory modules with GibbsModule: application to gene regulation in embryonic stem cells.</b> Dan Xie, Jun Cai, Na-Yu Chia, Huck H. Ng, Sheng Zhong.<br>
                                          <b><u>Genome Research</u></b>, 2008, 18:1325-1335. <a href="http://genome.cshlp.org/cgi/content/short/18/8/1325" target="_blank">Text</a>. <a href="GibbsModule/" target="_blank">Software</a>
                                    </li>
									
									<H4>Visualization and Personal Genome APP</H4>
                            		<li><b>GITAR: An Open Source Tool for Analysis and Visualization of Hi-C Data.</b>
                            			Riccardo Calandrelli, Qiuyang Wu, Jihong Guan, Sheng Zhong.<br>
                                    	<b><u>Genomics, Proteomics &amp; Bioinformatics</u></b>, 2018, 16(5):365-372.
                                          <a href="https://www.sciencedirect.com/science/article/pii/S1672022918304339?via%3Dihub" target="_blank">Text</a>,
                                          <a href="https://www.genomegitar.org/" target="_blank">Software</a>. 
                                    </li>

                            		
                            		<li><b>GIVE: portable genome browsers for personal websites.</b>
                            			Xiaoyi Cao, Zhangming Yan, Qiuyang Wu, Alvin Zheng, Sheng Zhong.<br>
                                    	<b><u>Genome Biology</u></b>, 2018, 19:92.
                                          <a href="https://doi.org/10.1186/s13059-018-1465-6" target="_blank">Text</a>,
                                          <a href="http://www.givengine.org/" target="_blank">Software</a>, 
                                          News &amp; Comments: <a href="https://www.nature.com/news/plot-a-course-through-the-genome-1.22553" target="_blank">Nature 549:117</a>, 
                                          Research Highlight: <a href="https://doi.org/10.1186/s13059-018-1470-9" target="_blank">Genome Biology 19:93 </a>, Technical feature: <a href="https://www.nature.com/articles/d41586-019-03705-y" target="_blank">Nature 576:171-172</a>.                             
                                    </li>

                            		
                            		<li><b>GeNemo: a search engine for web-based functional genomic data.</b>
                            			Yongqing Zhang, Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>Nucleic Acids Research</u></b>, 2016, 44: W122-W127.
                                          <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987887/" target="_blank">Text</a>,
                                          <a href="http://www.genemo.org/">Software</a><br>News
                                          coverage: <a href="http://hitconsultant.net/2016/05/04/33495/" target="_blank">
                                          HIT Consultant</a>,
                                          <a href="https://www.sciencedaily.com/releases/2016/05/160502093713.htm" target="_blank">
                                          Science Daily</a>,
                                          <a href="http://www.mediapost.com/publications/article/274766/meet-genemo-the-genomic-search-engine.html" target="_blank">
                                          MediaPost</a>,
                                          <a href="http://www.healthdatamanagement.com/news/new-ucsd-search-engine-can-sort-through-functional-genomics-data" target="_blank">
                                          HealthDataManagement</a>
                                    </li>

                                    <li><b>Enabling interspecies epigenomic comparison with CEpBrowser.</b>
                            			Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>Bioinformatics</u></b>, 2013, 29(9):1223–1225. <a href="https://academic.oup.com/bioinformatics/article/29/9/1223/219485" target="_blank">Text</a>. <a href="https://www.cepbrowser.org/" target="_blank">Software</a>
                                    </li>

                                    <li><b>Mapping personal functional data to personal genomes.</b>
                            			Marcelo Rivas-Astroza, Dan Xie, Xiaoyi Cao, Sheng Zhong.<br>
                                    	<b><u>Bioinformatics</u></b>, 2011, 27(24):3427-3429. <a href="http://bioinformatics.oxfordjournals.org/content/27/24/3427" target="_blank">Text</a>. <a href="perEditor/" target="_blank">Software</a>
                                    </li>

                                </ul>
                            </div>

                            <!--div role="tabpanel" class="tab-pane fade" id="Statistics-tab" aria-labelledby="Statistics-tab">
                                <ul class="text-left">
                            		
                                </ul>
                            </div-->

                            <!--div role="tabpanel" class="tab-pane fade" id="Software-tab" aria-labelledby="Software-tab">
                                <ul class="text-left">
                                </ul>
                            </div-->

                            <!--div role="tabpanel" class="tab-pane fade" id="Health-tab" aria-labelledby="Health-tab">
                                <ul class="text-left">
                                    <li><b>From Genomes to Societies: A Holistic View of Determinants of Human Health.</b>
                            		Yuyan Shi, Sheng Zhong.<br>
                                    <b><u>Current Opinion in Biotechnology</u></b>, 2014, 28:134-142. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24686286" target="_blank">Abstract</a>.
                                    </li>
                                </ul>
                            </div -->

                        </div>
                    </div>
                    
                    <!--br> 
                    
                    <div class="col-sm-12 wow fadeInDown">
                    	<h3 class="column-title">Latest bioRxiv papers</h3>
                    	<div class="panel-group" id="biorxiv" role="tablist" aria-multiselectable="true">

                        	<div class="panel panel-default">
                            	<div class="panel-heading" role="tab" id="biorxivHeadingZero">
                                	<h4 class="panel-title">
                                    <a class="collapsed" data-toggle="collapse" data-parent="#biorxiv" href="#biorxivCollapseZero" aria-expanded="false" aria-controls="biorxivCollapseZero">
                                        Dynamic changes in RNA-chromatin interactome promote endothelial dysfunction. </a>
                                	</h4>
                            	</div>
                            	<div id="biorxivCollapseZero" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingZero">
                                	<div class="panel-body">
                                    In endothelial cells (ECs) treated by high-glucose and TNFα, we employed single-cell RNA-sequencing and in situ mapping of RNA-genome interaction (iMARGI) assay to delineate temporal changes in transcriptome and RNA-chromatin interactome. 
                                    ECs displayed dramatic and heterogeneous changes in single cell transcriptome, accompanied by a dynamic and strong increase in inter-chromosomal RNA-DNA interactions, particularly among super enhancers. [<a href="https://doi.org/10.1101/712950" target="_blank">Inter-chromosomal RNA-DNA interactions, bioRxiv, 2019</a>].
                                	</div>
                            	</div>
                        	</div>
                    	</div>               
                	</div-->
                
                <!--div class="col-sm-12 wow fadeInDown">
                    <h3 class="column-title">Text book</h3>
                    <div class="panel-group" id="textbook" role="tablist" aria-multiselectable="true">
                        <div class="panel panel-default">
                            <div class="panel-heading" role="tab" id="bookHeadingOne">
                                <h4 class="panel-title">
                                    <a class="collapsed" data-toggle="collapse" data-parent="#textbook" href="#bookCollapseOne" aria-expanded="false" aria-controls="bookCollapseOne">
                                        3D Genome.
                                    </a>
                                </h4>
                            </div>
                            <div id="bookCollapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
                                <div class="panel-body">
                                    3D Genome: from technologies to visualization (draft). [<a href="https://zhonglab.gitbook.io/3dgenome/" target="_blank">ISBN: 987-1-17325643-0-5</a>]. 
                                    Textbook for <a href="http://systemsbio.ucsd.edu/beng183fa2020/", target="_blank">BENG183</a>. 
                                    Suggestions and content contributions are welcome!
                                </div>
                            </div>
                        </div>
                   	</div> 
        		</div--> <!-- class="col-sm-12 wow fadeInDown"--> 
    	</div> <!--/.container-->
    </section><!--/#portfolio-->

    <section id="services">
        <div class="container">
            <div class="section-header">
                <h2 class="section-title text-center wow fadeInDown">Software</h2>
                <p class="text-center wow fadeInDown"><a href="./softwarelist.html" target="_blank">Complete software list</a></p>
            </div>

            <div class="row">
                <div class="features">
                    
                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="0ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-bullseye"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"> <a href="https://zhong-lab-ucsd.github.io/GIVE_homepage/" target="_blank">GIVE</a></h4>
                                <p>Build your own genome browser website.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                    
                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="100ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-line-chart"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"> <a href="http://www.genemo.org/" target="_blank">GeNemo</a></h4>
                                <p>Internet search for genomic big data.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="200ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-cubes"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"> <a href="http://mariotools.ucsd.edu/" target="_blank">MARIO Tools</a></h4>
                                <p>Analyze RNA interaction data.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="300ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-bar-chart"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"><a href="http://www.cepbrowser.org/" target="_blank">CEpBrowser</a></h4>
                                <p>Comparative Epigenome Browser.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="400ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-language"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"><a href="http://systemsbio.ucsd.edu/perEditor/" target="_blank">perEditor</a></h4>
                                <p>Sequence mapping on personal genome.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                    <div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="500ms">
                        <div class="media service-box">
                            <div class="pull-left">
                                <i class="fa fa-pie-chart"></i>
                            </div>
                            <div class="media-body">
                                <h4 class="media-heading"><a href="GATE/" target="_blank">GATE</a></h4>
                                <p>Genome annotation using temporal epigenomic data.</p>
                            </div>
                        </div>
                    </div><!--/.col-md-4-->

                </div> <!-- /features -->
                   
            </div><!--/.row-->
            
            <!--div class="section-header">
                <p class="section-title text-center wow fadeInDown">Consortium web portal</p>
                <p class="text-center wow fadeInDown"><a href="./softwarelist.html" target="_blank">Complete software list</a></p>
        	</div-->
        	
        	<div class="row">
                <div class="features">
                	<h2 class="section-title text-center wow fadeInDown">4D Nucleome web portal</h2>

	        		<div class="col-md-4 col-sm-6 wow fadeInUp" data-wow-duration="300ms" data-wow-delay="600ms">
	                        <div class="media service-box">
	                            <div class="pull-left">
	                                <i class="fa fa-bullseye"></i>
	                            </div>
	                            <div class="media-body">
	                                <h4 class="media-heading"><a href="http://www.4dnucleome.org/" target="_blank">4D Nucleome Portal</a></h4>
	                                <p>Entry to NIH 4D Nucleome network.</p>
	                            </div>
	                        </div>
	             	</div><!--/.col-md-4-->

                 </div> <!-- /features -->
                             
            </div><!--/.row-->

        </div><!--/.container-->
    </section><!--/#services-->
    
    <section id="courselist">
        <div class="container">
            <div class="section-header">
                <br><br><br><br>
                <h2 class="section-title text-center wow fadeInDown">COURSES</h2>
	                 <div class="panel-group" id="teachingCourse" role="tablist" aria-multiselectable="true">
                        <div class="panel panel-default">
                            <div class="panel-heading" role="tab" id="courseHeadingOne">
                                <h4 class="panel-title">
                                    <a class="collapsed" data-toggle="collapse" data-parent="#teachingCourse" href="#courseCollapseOne" aria-expanded="false" aria-controls="courseCollapseOne">
                                        BENG100: Probability for Bioengineering
                                    </a>
                                </h4>
                            </div>
                            <div id="courseCollapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
                                <div class="panel-body">
                                    Go to <a href="http://systemsbio.ucsd.edu/beng100fa2020/" target="_blank">Fall 2020 course website</a>.
                                </div>
                            </div>
                        </div>
                        
                        <div class="panel panel-default">
                            <div class="panel-heading" role="tab" id="courseHeadingTwo">
                                <h4 class="panel-title">
                                    <a class="collapsed" data-toggle="collapse" data-parent="#teachingCourse" href="#courseCollapseTwo" aria-expanded="false" aria-controls="courseCollapseTwo">
                                        BENG183: Applied Genomic Technologies
                                    </a>
                                </h4>
                            </div>
                            <div id="courseCollapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
                                <div class="panel-body">
                                    Go to <a href="http://systemsbio.ucsd.edu/beng183fa2020/" target="_blank">Fall 2020 course website</a>.
                                </div>
                            </div>
                        </div>

                   	</div>    
	        </div>
        </div>
	</section> 
    
    <section id="get-in-touch">
        <div class="container">
            <div class="section-header">
                <h2 class="section-title text-center wow fadeInDown">Get in Touch</h2>
                <p class="text-center wow fadeInDown"> Powell-Focht Bioengineering Hall 371, University of California San Diego, 9500 Gilman Drive, MC 0412, La Jolla, CA 92093-0412<br> </p>
				<p class="text-center wow fadeInDown">Lab Phone: (858) 822-5649</p>
            </div>
        </div>
    </section><!--/#get-in-touch-->

			
     <section id="contact">
        <iframe id="google-map" style="height:650px;width:100%;border:none;" src="https://www.google.com/maps/embed/v1/place?key=AIzaSyBykXlm8AB6c85LDYieej0BYqD9loqw03Q&q=place_id:ChIJYyDpEsMG3IARlqX1UTBfFWk">
        <!-- div class="container-wrapper">
            <div class="container">
                <div class="row">
                    <div class="col-sm-4 col-sm-offset-8">
                        <div class="contact-form">
                            <h3>Contact us</h3>

                            <address>
                              <strong>Zhong lab</strong><br>
                              University of California San Diego
                            </address>

                            <form id="main-contact-form" name="contact-form" method="post" action="#">
                                <div class="form-group">
                                    <input type="text" name="name" class="form-control" placeholder="Name" required>
                                </div>
                                <div class="form-group">
                                    <input type="email" name="email" class="form-control" placeholder="Email" required>
                                </div>
                                <div class="form-group">
                                    <input type="text" name="subject" class="form-control" placeholder="Subject" required>
                                </div>
                                <div class="form-group">
                                    <textarea name="message" class="form-control" rows="8" placeholder="Message" required></textarea>
                                </div>
                                <button type="submit" class="btn btn-primary">Send Message</button>
                            </form>
                        </div>
                    </div>
                </div>
            </div -->
        </iframe>
    </section><!--/#bottom-->

    <script src="js/jquery.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="http://maps.google.com/maps/api/js?sensor=true"></script>
    <script src="js/owl.carousel.min.js"></script>
    <script src="js/mousescroll.js"></script>
    <script src="js/smoothscroll.js"></script>
    <script src="js/jquery.prettyPhoto.js"></script>
    <script src="js/jquery.isotope.min.js"></script>
    <script src="js/jquery.inview.min.js"></script>
    <script src="js/wow.min.js"></script>
    <script src="js/main.js"></script>
</body>
</html>
